_data/schemas/standard_administration.yml

```diff
@@ -0,0 +1,16 @@
+---
+administration: str()
+contact:
+  URL:
+    en: str(required=False)
+    fr: str(required=False)
+  email: str(required=False)
```

Comment:
I put required=False for the contact.email because as it currently stands, the standards that we have have this value set as null, which doesn't pass the schema validator. Will they be updated with actual emails, or is this the expected behaviour of all standards?

Reply:
I resolved by adding a template email address